### PR TITLE
Origin/feature/19 create car

### PR DESCRIPTION
**Pull request by Missael-JalaFund**
This pull request considers the construction of the car, and overriding method of the implemented interface.
Here you can see an image the result of the implementation

6966424945fc50221addd5a9a0fb9831

[Reference to Trello card](https://trello.com/c/Nc2ywgO9/12-create-car-functionality)

We suppress next warnings on linting:

Reason: We tried to make them self explanatory.
Elementes should be documented.{

We also solved Bug/Quad property change:
[Reference to Trello card](https://trello.com/c/bnwCjLhq/24-bug-quad-property-change)

### CODE REVIEW

File: DevVehicle35-Motors/App/CarInteraction.cs [hunk 1]
```diff
@@ -47,7 +47,7 @@ private void SelectNumberDoors()
             Console.WriteLine("======================================");
         }
 
-        private string DetermineColor(int optionColor) {
+        private static string DetermineColor(int optionColor) {
             if (optionColor == 1) {
```
Comment (CarlosZuritaRojas) [inline DevVehicle35-Motors/App/CarInteraction.cs:51]: To maintain good practices based on styleCop and C# we should keep braces on their own lines of code.

Comment (CarlosZuritaRojas) [reply DevVehicle35-Motors/App/CarInteraction.cs:51]: In line 58, you might compare option color == 4.

File: DevVehicle35-Motors/App/CarInteraction.cs [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        {
+            Console.WriteLine("======= Select Number of Doors =======================");
+            Console.WriteLine("======================================");
+            Console.WriteLine("1. 2 Doors");
+            Console.WriteLine("2. 4 Doors");
+            Console.WriteLine("======================================");
+        }
+
+        private static string DetermineColor(int optionColor) {
+            if (optionColor == 1) {
```
Comment (RaulCFunJala) [inline DevVehicle35-Motors/App/CarInteraction.cs:51]: Think here a switch statement its simpler and better than have to much if-else.

**Review by RaulCFunJala — CHANGES_REQUESTED** [review]

File: DevVehicle35-Motors/App/CarInteraction.cs (hunk 2 above)
Comment (alejandrozeb) [reply DevVehicle35-Motors/App/CarInteraction.cs:52]: I thing switch statement corrompes single responsibility principle thats why I prefer ifelse than switch statement in my style code.

**Review by RaulCFunJala — APPROVED** [review]

**Review by CarlosZuritaRojas — APPROVED** [review]

File: DevVehicle35-Motors/App/CarInteraction.cs [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+                return "Orange";
+            } else if (optionColor == 4)
+            {
+                return "White";
+            }
+
+            return "Red";
+        }
+
+        private static int DetermineNumberDoors(int optionColor)
```
Comment (ElizabethMamani) [inline DevVehicle35-Motors/App/CarInteraction.cs:67]: I understand that the "optionColor" variable in this function refers to the number of car doors, but visually it confuses me a little.

Comment (alejandrozeb) [reply DevVehicle35-Motors/App/CarInteraction.cs]: Thanks! variable already changed for optionDoors.

**Review by ElizabethMamani — APPROVED** [review]

**Review by andreapenaranda — APPROVED** [review]

**Review by kevil17 — COMMENTED** [review]
ok

**Pull request merged**